dispatch/dispatch.go

```diff
@@ -362,6 +364,10 @@ func (ag *aggrGroup) fingerprint() model.Fingerprint {
 	return ag.labels.Fingerprint()
 }
 
+func (ag *aggrGroup) GroupKey() uint64 {
+	return uint64(ag.labels.Fingerprint() ^ ag.routeFP)
```

Comment:
Maybe I'm missing something, but we can just keep the `Fingerprint` type here all the way no? Then we don't have to do any casting along the way.